resources/tests.mjs

```diff
@@ -3,6 +3,21 @@ import { BenchmarkTestStep } from "./benchmark-runner.mjs";
 const numberOfItemsToAdd = 100;
 export const Suites = [];
 
+Suites.enable = function (names) {
+    this.forEach((suite) => {
+        suite.disabled = true;
```

Comment:
It doesn't seem like this forEach is needed. We can just add else clause in the forEach call below.

Reply:
DONE